packages/graphql-typescript-definitions/src/index.ts

```diff
@@ -1,7 +1,5 @@
 import {EventEmitter} from 'events';
```

Comment:
I still see one reference in the README to passing an explicit list of globs to the CLI — we should get rid of that, right?